requestbuilder/command.py

```diff
@@ -36,6 +36,9 @@
 from requestbuilder.util import add_default_routes, aggregate_subclass_fields
 
 
+import six
```

Comment:
Per PEP 8, third party library imports come between standard library imports and local imports.